src/main/configurations/Translate/Configuration_Zaken_PostZgwZaakInformatieObject.xml

```diff
@@ -14,14 +14,23 @@
                 <Exit name="EXCEPTION" state="ERROR" />
             </Exits>
 
+            <PutInSessionPipe name="getInhoudParameter">
+                <Param name="ref_inhoud" sessionKey="inhoud" />
+                <Forward name="success" path="ValidatePost" />
+            </PutInSessionPipe>
+            <Json2XmlValidatorPipe name="ValidatePost"
+                schema="Zgw/Documenten/Model/PostZgwEnkelvoudigInformatieObject.xsd"
+                root="ZgwEnkelvoudigInformatieObject"
+                outputFormat="JSON"
+                deepSearch="true"
+                throwException="true">
+                <Forward name="success" path="SubstituteDocumentDataRef" />
+                <Forward name="exception" path="buildErrorMsg" />
+            </Json2XmlValidatorPipe>
+            <EchoPipe name="SubstituteDocumentDataRef" restoreMovedElements="true">
+                <Forward name="success" path="PostZgwEnkelvoudigInformatieObjectSender"/>
+            </EchoPipe>
             <SenderPipe name="PostZgwEnkelvoudigInformatieObjectSender">
-                <Json2XmlInputValidator name="ValidatePost"
-                    schema="Zgw/Documenten/Model/PostZgwEnkelvoudigInformatieObject.xsd"
-                    root="ZgwEnkelvoudigInformatieObject"
-                    outputFormat="JSON"
-                    deepSearch="true"
-                    throwException="true"
-                />
                 <HttpSender name="PostZgwEnkelvoudigInformatieObjectHttpSender"
                     methodType="POST"
                     headersParams="Authorization,Accept-Crs,Content-Crs,Accept"
```

Comment:
Same for this one. Is it possible to merge it? Can the restoreMovedElements attribute be on SenderPipe?

Reply:
Met deze wijziging werkt hij niet meer. Ik zie een fout in SoapUI. In Ladybug zie je dat de HttpSender als input de message zonder de base64 bytes krijgt. Het terugzetten van de session keys gebeurt blijkbaar na het verzenden - te laat.

![image](https://github.com/ibissource/zaakbrug/assets/6784732/d1d0569d-d0cb-4b07-8bdf-3fc74300c0ab)